### PR TITLE
🌐 Add Chinese translation for FastAPI People

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by waynerv** [comment]
sections -> "栏目", thresholds -> "统计阈值"

**Comment by waynerv** [comment]
Let me know when you finished,I'll review it.

File: docs/zh/docs/fastapi-people.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+**FastAPI** 得到了社区的大力支持。我想强调他们的贡献。
+
+这些人：
+
+* [帮助他人解决 GitHub 的 issues](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}。
+* [创建 Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}。
+* 审核 Pull Requests， 对于 [翻译](contributing.md#translations){.internal-link target=_blank} 尤为重要。
+
+一阵热烈的掌声。 👏 🙇
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:35]:
```suggestion
向他们致以掌声。 👏 🙇
```

File: docs/zh/docs/fastapi-people.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+{% endfor %}
+
+</div>
+{% endif %}
+
+## 杰出的贡献者
+
+以下是 **杰出的贡献者**。 👷
+
+这些用户积极 [创建 Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}。
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:73]:
```suggestion
这些用户 [创建了最多已被合并的 Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}。
```

File: docs/zh/docs/fastapi-people.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+还有很多其他贡献者(超过100个)，你可以在 <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">FastAPI GitHub 贡献者页面</a> 中看到他们。👷
+
+## 杰出的审核者
+
+以下用户是「杰出的审核者」。 🕵️
+
+### 翻译审核
+
+我只会说少数几种语言。所以这些审核者有权力批准 [文档翻译](contributing.md#translations){.internal-link target=_blank}。如果没有它们，就不会有多语言文档。
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:95]:
```suggestion
我只会说少数几种语言（而且还不是很流利 😅）。所以，具备[能力去批准文档翻译](contributing.md#translations){.internal-link target=_blank} 是这些评审者们。如果没有它们，就不会有多语言文档。
```

File: docs/zh/docs/fastapi-people.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+{% endfor %}
+
+</div>
+{% endif %}
+
+还有很多其他贡献者(超过100个)，你可以在 <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">FastAPI GitHub 贡献者页面</a> 中看到他们。👷
+
+## 杰出的审核者
+
+以下用户是「杰出的审核者」。 🕵️
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:91]:
```suggestion
以下用户是「杰出的评审者」。 🕵️
```

File: docs/zh/docs/fastapi-people.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+### 翻译审核
+
+我只会说少数几种语言。所以这些审核者有权力批准 [文档翻译](contributing.md#translations){.internal-link target=_blank}。如果没有它们，就不会有多语言文档。
+
+---
+
+**杰出的审核者** 🕵️ 已评估大量的 Pull Requests，能保证了代码，文档，尤其是 **翻译** 的质量。
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:99]:
```suggestion
**杰出的评审者** 🕵️ 评审了最多来自他人的 Pull Requests，他们保证了代码、文档尤其是 **翻译** 的质量。
```

File: docs/zh/docs/fastapi-people.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## 关于数据 - 技术细节
+
+该页面的目的是强调社区为帮助他人而付出的努力。
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:157]:
```suggestion
该页面的目的是突出社区为帮助他人而付出的努力。
```

File: docs/zh/docs/fastapi-people.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+尤其是那些不引人注目且涉及更困难的任务，例如帮助他人解决问题或者审核翻译请求。
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:159]:
```suggestion
尤其是那些不引人注目且涉及更困难的任务，例如帮助他人解决问题或者评审翻译 Pull Requests。
```

File: docs/zh/docs/fastapi-people.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+
+该数据是每个月计算的，您可以阅读 <a href="https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py" class="external-link" target="_blank">源代码</a>。
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:161]:
```suggestion
该数据每月计算一次，您可以阅读 <a href="https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py" class="external-link" target="_blank">源代码</a>。
```

File: docs/zh/docs/fastapi-people.md (hunk 4 above)
Comment (waynerv) [reply docs/zh/docs/fastapi-people.md:91]: code review normally translated to "代码评审"

File: docs/zh/docs/fastapi-people.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+<div class="user-list user-list-center">
+{% for user in people.experts %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## 杰出的贡献者
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:69]:
```suggestion
## 杰出贡献者
```

File: docs/zh/docs/fastapi-people.md [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Pull Requests: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+还有很多其他贡献者(超过100个)，你可以在 <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">FastAPI GitHub 贡献者页面</a> 中看到他们。👷
+
+## 杰出的审核者
```
Comment (waynerv) [inline docs/zh/docs/fastapi-people.md:89]:
```suggestion
## 杰出审核者
```

**Review by waynerv — CHANGES_REQUESTED** [review]

File: docs/zh/docs/fastapi-people.md [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+<div class="user-list user-list-center">
+{% for user in people.top_contributors %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Pull Requests: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+还有很多其他贡献者(超过100个)，你可以在 <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">FastAPI GitHub 贡献者页面</a> 中看到他们。👷
```
Comment (RunningIkkyu) [inline docs/zh/docs/fastapi-people.md:87]: Use chinese bracket here.

```
还有很多其他贡献者（超过100个）
```

**Review by RunningIkkyu — CHANGES_REQUESTED** [review]

**Review by waynerv — APPROVED** [review]
LGTM

**Review by Laineyzhang55 — APPROVED** [review]

**Comment by waynerv** [comment]
Why closed? This pr will be merged in next release.

**Comment by hareru** [comment]
Sorry, I executed a wrong command.

**Comment by tiangolo** [comment]
Awesome, thanks for your contribution @hareru ! :rocket: 

And thanks for the reviews @RunningIkkyu, @waynerv, and @Laineyzhang55 :bow: :coffee:

**Pull request merged**